presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestMaterializedViewQueryOptimizer.java

```diff
@@ -1140,6 +1140,62 @@ public void subqueryAggregationSupportedFunction()
                 BASE_TABLE_2, ImmutableMap.of(VIEW_2, viewSql2)));
     }
 
+    @Test
+    public void testAvgRewriteWithSumAndCount()
+    {
```

Comment:
I wrote the test with the assumption we fail- given I'm currently explicitly failing in case of `expressions.size() > 1 || (expressions.get(0) instanceof Identifier)`.

Reply:
Yes